generators/templates/package.json

```diff
@@ -6,9 +6,16 @@
   },
   "version": "<%= pfelementVersion %>",
   "description": "<%= readmeName %> element for PatternFly Elements",
+  "keywords": [
+    "web-components",
+    "html"
+  ],
   "repository": {
     "type": "git",
+    <% if (isPfelement) { %>"url": "github:patternfly/patternfly-elements",
```

Comment:
Be sure to use a line-slurp here, the spacing is coming out wonky in the output file.  The readmeName doesn't seem to be working either.

Reply:
I think I fixed this in my latest commit.

Reply:
I didn't see any issues with the README though. I have tested this both in and out of the PatternFly Elements repository without any issues.

Reply:
Twas my poor naming of the element - closing this out and approving.